### PR TITLE
Ensures the cfn-hup service is executable

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

File: AWSutils.sh [hunk 1]
```diff
@@ -350,6 +350,10 @@ function InstallCfnBootstrap {
       chroot "${CHROOTMNT}" ln -sf /opt/aws/apitools/cfn-init/init/redhat/cfn-hup /etc/init.d/cfn-hup || \
          err_exit "Failed creating symlink for cfn-hup service"
 
+      err_exit "Making sure cfn-hup service is executable..." NONE
+      chroot "${CHROOTMNT}" chmod +x /etc/init.d/cfn-hup || \
```
Comment (ferricoxide) [inline AWSutils.sh:354]: Marginally less expensive to just do:
~~~
      chmod +x "${CHROOTMNT}/etc/init.d/cfn-hup" || \
~~~

Comment (lorengordon) [reply AWSutils.sh]: ok done!

**Review by ferricoxide — APPROVED** [review]
Looks trivial (and shellcheck greenlit it): "approved"

**Comment by evandena** [comment]
Is this why my recent SPEL build RHEL8 build might be failing?

`amazon-ebs.minimal-rhel-8-hvm: <10>Aug 19 15:19:07 AWSutils.sh[105331]: Making sure cfn-hup service is executable...
    amazon-ebs.minimal-rhel-8-hvm: + [[ NONE =~ ^[0-9]+$ ]]
    amazon-ebs.minimal-rhel-8-hvm: + chmod +x /mnt/ec2-root/etc/init.d/cfn-hup
    amazon-ebs.minimal-rhel-8-hvm: chmod: cannot operate on dangling symlink '/mnt/ec2-root/etc/init.d/cfn-hup'
    amazon-ebs.minimal-rhel-8-hvm: + err_exit 'Failed making cfn-hup service executable'
    amazon-ebs.minimal-rhel-8-hvm: + local ERRSTR
    amazon-ebs.minimal-rhel-8-hvm: + local ISNUM
    amazon-ebs.minimal-rhel-8-hvm: + local SCRIPTEXIT
    amazon-ebs.minimal-rhel-8-hvm: + ERRSTR='Failed making cfn-hup service executable'
    amazon-ebs.minimal-rhel-8-hvm: + ISNUM='^[0-9]+$'
    amazon-ebs.minimal-rhel-8-hvm: + SCRIPTEXIT=1
    amazon-ebs.minimal-rhel-8-hvm: + [[ true == true ]]
    amazon-ebs.minimal-rhel-8-hvm: + logger -i -t AWSutils.sh -p kern.crit -s -- 'Failed making cfn-hup service executable'
    amazon-ebs.minimal-rhel-8-hvm: <10>Aug 19 15:19:07 AWSutils.sh[105333]: Failed making cfn-hup service executable`

**Comment by lorengordon** [comment]
@evandena I've started another CI build to confirm, and created an issue #60. We'll get this fixed.

**Pull request merged**